openarc/openarc.c

```diff
@@ -3533,6 +3537,14 @@ mlfi_eom(SMFICTX *ctx)
 
 						arc_set_cv(afc->mctx_arcmsg,
 						           cv);
+
+						continue;
```

Comment:
What is this for?  I believe there can be only one chain result extracted from our own A-R fields.